app/models/order.rb

```diff
@@ -96,7 +96,7 @@ def business_days_after num
 
     def delivery_day_limit
       errors.add :delivery_day, :blank and return if delivery_day.blank?
-      if delivery_day < min_day or delivery_day > max_day or [0, 6].include?(delivery_day.wday)
+      if delivery_day < min_day or max_day < delivery_day or [0, 6].include?(delivery_day.wday)
```

Comment:
[rubocop] Use a guard clause instead of wrapping the code inside a conditional expression.